Woof/Woof/ViewModel/SitterProfileViewModel.swift

```diff
@@ -65,4 +70,12 @@ final class SitterProfileViewModel: ObservableObject {
 
         return sitter
     }
+
+    private func restoreOriginalValues(_ currentSitter: Sitter) {
```

Comment:
Can we use this function in the init? DRY

Reply:
Fixed.